ansible/plugins/inventory/nodejs_yaml.py

```diff
@@ -138,21 +138,27 @@ def get_secrets_path():
             return path
         else:
             print("WARNING: NODE_BUILD_SECRETS defined but not a directory", file=sys.stderr)
+            print("It must be the path to a local checkout of https://github.com/nodejs-private/secrets", file=sys.stderr)
             return None
 
     path = os.path.realpath('../../secrets/build/')
     if os.path.isdir(path):
         return path
 
+    print("WARNING: ../../secrets/build/ is not a directory", file=sys.stderr)
```

Comment:
I'm not sure about these 3 warnings, it means that if you have it in the last checked location you'll see two of them, but that location isn't a problem so doesn't deserve to trigger warnings.
Maybe these should be printed at the end, "I checked here, here and here but didn't find it".